include/xtensor/xstrided_view.hpp

```diff
@@ -423,6 +426,12 @@ namespace xt
     }
     //@}
 
+    template <class CT, class S, layout_type L, class FST>
+    inline auto xstrided_view<CT, S, L, FST>::set_offset(size_type offset) -> void
+    {
+        base_type::set_offset(offset);
+    }
```

Comment:
This can be removed after replacing the declaration with a using statement (see previous remark)